dist/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html ng-app="BlocChat">
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Bloc Chat</title>
+    <link rel="stylesheet" href="/styles/style.css">
+</head>
+<body>
+	<div class="side-bar">
+		<div class="chat-title">
+			<strong>Bloc Chat</strong>
+		</div>
+		<div class="room-list">
+			<ui-view></ui-view>
+		</div>
+	</div>
+
+
+
+
+<script src="/scripts/app.js"></script>
+ <!-- Angular -->
+ <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular.min.js"></script>
+ <!-- Firebase -->
+<script src="https://www.gstatic.com/firebasejs/3.3.0/firebase.js"></script>
+<!-- AngularFire -->
+<script src="https://cdn.firebase.com/libs/angularfire/2.0.1/angularfire.min.js"></script>
+<script>
```

Comment:
This is in the wrong file, the dist folder is what is compiled from a grunt build. Remind to me to talk about this tomorrow.